src/pages/MyTokens/NFTPage.tsx

```diff
@@ -108,7 +108,10 @@ export const NFTPage = () => {
       await fetch({ page: 1, pageSize, seller: selectedAccount?.address });
       const _tokens = await api.nft?.getAccountMarketableTokens(selectedAccount.address) as NFTToken[];
 
-      setTokens(_tokens);
+      setTokens(await Promise.all(_tokens.map(async (token) => ({
```

Comment:
:warning: 
Make sure to remove merge of NFTToken and NFTCollection from code. We should use collections/tokens separately now and pass as two different props, not as merge. OTherwise code is unreadable and rly hard to support